### PR TITLE
Fix message on switching namespace to properly show new namespace

### PR DESCRIPTION
**Pull request by jinnovation**
The namespace-change callback messages as currently implemented display the current namespace rather than the post-change namespace.

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ versioning][semver].
 
 ## Upcoming
 
+### Fixed
+
+- Fixed bug where log message on changing namespace did not properly reflect the selected namespace.
```
Comment (jinnovation) [inline docs/CHANGELOG.md:16]:
```suggestion
- Fixed bug where log message on changing namespace did not properly reflect the selected namespace. (#297)
```

**Comment by noorul** [comment]
conflict

**Review by noorul — APPROVED** [review]
LGTM

**Comment by noorul** [comment]
conflict

**Pull request merged**